src/screens/FeedScreen/Content.js

```diff
@@ -71,14 +70,23 @@ const Content = ({
     item.images_url,
     message
   );
+
+  const isBlurredPost = item?.isBlurredPost;
+
   const calculateMaxLine = () => {
+    if (isBlurredPost && images_url.length > 0) {
+      return 1;
+    }
     if (item.post_type === POST_TYPE_LINK || images_url.length > 0) {
       return 5;
     }
     if (item.post_type === POST_TYPE_POLL) {
       const result = Math.round((layoutHeight - heightPoll - heightTopic) / lineHeight);
-
-      return result >= 0 ? result : 0;
+      const isHaveTopic = topics.length > 0 ? 3 : 2;
+      if (item.pollOptions.length < 4) {
+        return result >= 0 ? result - 1 : 0;
+      }
+      return isHaveTopic;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for calculating the maximum line (`calculateMaxLine`) has been changed significantly. It now takes into account whether the post is blurred and if it contains images. This seems to be a reasonable change, but it's important to ensure that this new logic doesn't introduce any unexpected behavior in different scenarios.



<!-- This is an auto-generated comment by OSS CodeRabbit -->